src/Okologiskegrunnkart/FeatureInfo/PolygonElement.js

```diff
@@ -44,7 +44,7 @@ const PolygonElement = ({
     <div className="generic_element">
       <ListItem
         id="polygon-element-list"
-        button={numberResults > 0 ? true : false}
```

Comment:
A bit of redundancy here mapping true to true and false to false

Reply:
Totally agree, that's why I removed it :D replaced it with `button={numberResults}`